test/ResourcesClassLibraryNoAttribute/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-*",
+  "version": "1.1.0-*",
```

Comment:
We had an offline conversation about this part of the PR. You're making these two projects inconsistent with test projects elsewhere. Test projects (well, those few that require versions) should stick with `1.0.0`.

Please revert all `project.json` updates except the deletion in `samples/LocalizationSample/project.json`.